main.go

```diff
@@ -87,7 +91,6 @@ func run(ctx context.Context) error {
 			return err
 		}
 
-		otelShutdown(ctx)
 		log.Info(ctx, "Shutdown complete")
```

Comment:
I don't understand this change.

Reply:
This got moved up to line 65 - it's called when the service shuts down and is otel specific. This ensures the service closes down nicely.